.travis.yml

```diff
@@ -10,16 +10,8 @@ matrix:
   - python: "3.6"
     env: TEST_SUITE=lint
 install:
-    # Install codecov, the library for the code coverage reporting tool we use
-  - pip install codecov
-
-    # Install pycodestyle, the library for checking pep8 code conformity.
-  - pip install pycodestyle==2.3.1
-
-    # Install all API packages
-  - pip install ./zulip
-  - pip install ./zulip_bots
-  - pip install ./zulip_botserver
+  - tools/provision
+  - source "zulip-api-py2-venv/bin/activate"
```

Comment:
I like the idea of doing `provision` in the tests here (though be careful not to end up hardcoding the Python version to 2 by accident!), but since we don't actually need codecov in development, we should probably leave `pip install codecov` here.